dace/codegen/targets/intel_fpga.py

```diff
@@ -562,6 +562,11 @@ def generate_module(self, sdfg, state, kernel_name, module_name, subgraph, param
             if isinstance(p, dace.data.View):
                 continue
             arg = self.make_kernel_argument(p, pname, is_output, True)
+            
+            #change c type long long to opencl type long
+            if arg.__contains__("long long"):
```

Comment:
same comment as before.

Also note that arg can be none (see the if two lines below)